src/utils.js

```diff
@@ -1,5 +1,36 @@
 import dayjs from 'dayjs';
-import { Type, Destination, Filter, Sorting } from './enums';
+import { RenderPosition, Type, Destination, Filter, Sorting } from './enums';
+
+const renderTemplate = (container, template, place) => {
+  container.insertAdjacentHTML(place, template);
+};
+
+const render = (container, element, place) => {
+  switch (place) {
+    case RenderPosition.AFTERBEGIN:
+      container.prepend(element);
+      break;
```

Comment:
```suggestion
    case RenderPosition.AFTERBEGIN: {
      container.prepend(element);
      break;
    }
```
скобки - обязательно. Добавить на все

Reply:
Исправила